cluster/kubemark/pre-existing/config-default.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
@shyamjvs What are you thoughts about adding a README in this directory?  Or where would you recommend documenting how to using the pre-existing provider?